HelpMyStreetFE/HelpMyStreetFE/Helpers/SupportActivityHelpers.cs

```diff
@@ -22,6 +22,10 @@ public static string Icon(this SupportActivities activity)
                 SupportActivities.WellbeingPackage => "package.svg",
                 SupportActivities.CommunityConnector => "phone-green.svg",
                 SupportActivities.ColdWeatherArmy => "snowflake.svg",
+                SupportActivities.MealsToYourDoor => "mealsonwheels.svg",
```

Comment:
are you happy for this to exist or should it be renamed to mealstoyourdoor.png

Reply:
Updated